### ISSUE TITLE
Address breaking linting errors in cra- folders

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria#4137]

**Comment by davidradl** [comment]
As part of this change to use create react app, I found that we are not handling images consistently across the production and development builds. The reason was that we build with the homepage as ".", so we try to load the tenanted images. But when the current directory is a subfolder it does not account for that subfolder in the minified versions of the resources.


What I have changed it to is that we get the images and javascript from non-tenanted static folder.  This works but it might not be what we want. so there are 2 options:

1) Use non tenanted resources. If the use case is that a presentation server will only run one tenant, and a user will never change tenant, then tenant specific changes could be made in the static folder and picked up with a rebuild. We have not got a design to pick up a tenant specific authentications.  

2) If the use case is that we need to handle aspects of multi tenants at runtime; e.g. have 2 sets of static resource one for tenant1 and one for tenant2 and choose between then at runtime; we would have the tenanted url come in and then need to do server side url rewriting; this would need to be investigated. Ideally we would want to look for a tenant specific image then if that is not found then default to the standard one. I am told that we would need to be careful of webpack caching.

Option 1 would be pragmatic and allow us to move forward and move the ui out of the repo. We could implement the preferable option 2 in the future.
  
@mandy-chessell @jmmadsen @sarbull  @grahamwallis @planetf1
